Scripts/Tracking/Query/PointingQuery.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace VRTK.Core.Tracking.Query
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using VRTK.Core.Process;
+
+    /// <summary>
+    /// Determines whether a <see cref="GameObject"/> points at another <see cref="GameObject"/>.
+    /// </summary>
+    /// <remarks>
+    /// No physics checks are done and as such occlusion isn't part of the information gained by this component.
+    /// </remarks>
+    public class PointingQuery : MonoBehaviour, IProcessable
+    {
+        /// <summary>
+        /// The object that will be checked for pointing at <see cref="target"/>.
+        /// </summary>
+        [Tooltip("The object that will be checked for pointing at target.")]
```

Comment:
`The object that will be checked to see if the <see cref="source"/> is pointing at it.`